src/directives/dir/dir.ts

```diff
@@ -0,0 +1,31 @@
+import {Directive, HostBinding, EventEmitter, Output, Input} from 'angular2/core';
+import {OneOf} from "../../core/annotations/one-of";
+
+
+/**
+ * Directive to listen to changes of direction of part of the DOM.
+ *
+ * Applications should use this directive instead of the native attribute so that Material
+ * components can listen on changes of direction.
+ */
+@Directive({
+  selector: '[dir]',
+  exportAs: '$implicit'
```

Comment:
I think making `dir` the `$implicit` is too pushy, since `dir` is something you could throw on anything. Maybe export as `$dir` ?